README.md

```diff
@@ -227,7 +227,13 @@ Future work and Limitations
 Change Log (From version 2.2.0 and onwards)
 ==============
 
-### __2.6.0-SNAPSHOT__
+### __3.0.0__
+
+* Upgraded to Hive 1.2.1 (Note: new major release with backwards incompatibility issues). As of Hive 1.2 there are a number of new reserved keywords, see [DDL manual](https://cwiki.apache.org/confluence/display/Hive/LanguageManual+DDL#LanguageManualDDL-Keywords,Non-reservedKeywordsandReservedKeywords) for more information. 
+If you happen to have one of these as an identifier, you could either backtick quote them (e.g. \`date\`, \`timestamp\` or \`update\`) or set hive.support.sql11.reserved.keywords=false.                                            
+* Removed the custom HiveConf hive.vs. Use hadoop.tmp.dir instead.   
```

Comment:
Maybe hiverunner should ship without bundling any dependencies? Let's see how much of a hassle it is if you set the dependencies to provided. We would have to add a guide on what to include, but the chance would be higher that each user would find a set of dependencies that would work with his/hers project.

We should probably create a small dummy reference project with branches for different versions of the pom as an example and documentation as well as an integration test.